src/emit.ts

```diff
@@ -64,4 +65,19 @@ export const recordEvent = (
 
   // Record the event message sent by the emit
   events[cid][event].push(args)
+
+  // see if we are emitting a v-model syntax sugar update:prop event
```

Comment:
nit: maybe add a comment to explain that, if that's the case, we want to update props automatically.